src/m_alloc.cpp

```diff
@@ -40,6 +40,9 @@
 #include <malloc/malloc.h>
 #elif defined(__OpenBSD__)
 #include <stdlib.h>
+#elif defined(__APPLE__)
+#include <stdlib.h>
+#include <malloc.h>
 #else
```

Comment:
How do changes Haiku relate to macOS?

Reply:
No idea how it got into this PR. This is not needed.